api_app/ingestors_manager/ingestors/malware_bazaar.py

```diff
@@ -0,0 +1,135 @@
+import io
+import logging
+from datetime import datetime
+from typing import Any, Iterable
+from unittest.mock import patch
+
+import pyzipper
+import requests
+
+from api_app.ingestors_manager.classes import Ingestor
+from api_app.ingestors_manager.exceptions import IngestorRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections
+
+logger = logging.getLogger(__name__)
+
+
+class MalwareBazaar(Ingestor):
+    url: str
+    hours: int
+    signatures: str
```

Comment:
if the `Parameter` called `signatures` it is actually a secret, the `config()` function would have assigned it to `_signatures` in this class.
Idk how this is working on your side